src/ublox.cpp

```diff
@@ -1,19 +1,25 @@
 #include "ublox.h"
 
-#define DEG2RAD (3.14159 / 180.0)
+#include <time.h>
 
-UBLOX* GPSptr;
+//#include "printf.h"
+//#define DBG(...) printf(__VA_ARGS__)
+#define DBG(...)
```

Comment:
I think we should just remove all instances of `DBG`

Reply:
Done.